### PR TITLE
Subscription changes

### PR DESCRIPTION
**Pull request by jbrooksuk**
Making changing to subscribers:

1. Dashboard shows what (if any) subscriptions someone has.
2. Adding a link in the subscriber email to show subscriptions.

### CODE REVIEW

**Comment by joecohens** [comment]
Very nice!

File: app/Models/Subscriber.php [hunk 1]
```diff
@@ -67,7 +74,7 @@ public static function boot()
      *
      * @return \Illuminate\Database\Eloquent\Relations\HasMany
      */
-    public function subcriptions()
+    public function subscriptions()
```
Comment (joecohens) [inline app/Models/Subscriber.php]: lol

**Comment by jbrooksuk** [comment]
Just the email link left to do for now.

File: app/Models/Subscriber.php (hunk 1 above)
Comment (GrahamCampbell) [inline app/Models/Subscriber.php]: Can we have the typos and fixes merged seperately please so we can ship them with 2.1.1, and have the feature part going into 2.2.0.

**Comment by jbrooksuk** [comment]
We still need a place for subscribers to manage their subscriptions from, also the unsubscribe link.

**Comment by jbrooksuk** [comment]
I forgot I did the manage page :) just the unsub link.

**Comment by GrahamCampbell** [comment]
I've just rebased this. Pending final review.

**Comment by jbrooksuk** [comment]
Happy if @joecohens is?

**Comment by joecohens** [comment]
:beers:

**Comment by jbrooksuk** [comment]
WOOP!

**Pull request merged**